client/app.json

```diff
@@ -31,12 +31,13 @@
       "expo-router",
       [
         "expo-asset",
-  
         {
-          "assets": ["./assets/images/bear.gif"]
+          "assets": [
+            "./assets/images/bear.gif"
+          ]
```

Comment:
do we need to add the other ones here?

Reply:
Can you check if we need that at all there? Because i think we're able to access all the images regardless

Reply:
I believe this is unneccesary. I'm using assets in my current feature, so i will review and make edits there :)